src/hal/src/lib.rs

```diff
@@ -201,7 +201,13 @@ bitflags! {
         const SAMPLER_MIP_LOD_BIAS = 0x0008 << 64;
 
         /// Make the NDC coordinate system pointing Y up, to match D3D and Metal.
-        const NDC_Y_UP = 0x01 << 80;
+        const NDC_Y_UP = 0x0001 << 80;
+        /// Allow indexing sampled texture descriptor arrays with dynamically non-uniform data
+        const SAMPLED_TEXTURE_DESCRIPTOR_INDEXING = 0x0002 << 80;
```

Comment:
these should be in the `CORE_MASK`, actually, since they correspond directly to Vulkan extensions/features.
To clarify, the "core" is slightly misleading here. It was meant to be a mask of things that come from Vulkan.